### ISSUE TITLE
Only works with unencrypted bridges

### ISSUE THREAD

**Issue by brknkfr** [Liarra/signal-wa-double-bridging#1]
First, thank you very much for this great tutorial. It seems still to work with the current bridges, but ...

... please mention somewhere in the beginning, that this works only with _**unencrypted**_ bridges!

(I got locked out from WhatsApp, because the signal bridge and whatsapp bridge played ping-pong with messages like "I don't understand you, I don't bridge your message" / "No, I don't understand _you_, I won't bridge _your_ message." / And so on ...)

**Comment by Liarra** [comment]
Hello! Thanks for the report, and sorry for the late reply. Just to clarify, before the ping-pong exchange occurred, did you enable end-to-bridge encryption in your setup as explained here (https://docs.mau.fi/bridges/general/end-to-bridge-encryption.html)? Or is there some other potential encryption-related config combo that we should mention?

**Comment by brknkfr** [comment]
Yes. The end-to-bridge-encryption was enabled before trying to set up a whatsapp-signal-bridged room.
